
[orcid.org/0000-0001-6920-0041](https://orcid.org/0000-0001-6920-0041)

<div id="refs" class="references csl-bib-body hanging-indent"
line-spacing="2">

<div id="ref-ogradyetal2025communicationsbiology" class="csl-entry">

O’Grady, J. F., **McHugo, G. P.**, Ward, J. A., Hall, T. J., Faherty
O’Donnell, S. L., Correia, C. N., Browne, J. A., McDonald, M., Gormley,
E., Riggio, V., Prendergast, J. G. D., Clark, E. L., Pausch, H., Meade,
K. G., Gormley, I. C., Gordon, S. V., & MacHugh, D. E. (2025).
Integrative genomics sheds light on the immunogenetics of tuberculosis
in cattle. *Communications Biology*, *8*(1), 479.
doi:[10.1038/s42003-025-07846-x](https://doi.org/10.1038/s42003-025-07846-x).

</div>

<div id="ref-mchugoetal2025biorxiv" class="csl-entry">

**McHugo, G. P.**, Ward, J. A., Browne, J. A., O’Gorman, G. M., Meade, K.
G., Hill, E. W., Hall, T. J., & MacHugh, D. E. (2025). *Functional
genomics of trypanotolerant and trypanosusceptible cattle infected with
Trypanosoma congolense across multiple time points and tissues*
\[Preprint\]. bioRxiv, 2025.01.31.635858.
doi:[10.1101/2025.01.31.635858](https://doi.org/10.1101/2025.01.31.635858).

</div>

<div id="ref-mchugoetal2025heredity" class="csl-entry">

**McHugo, G. P.**, Ward, J. A., Ng’ang’a, S. I., Frantz, L. A. F.,
Salter-Townshend, M., Hill, E. W., O’Gorman, G. M., Meade, K. G., Hall,
T. J., & MacHugh, D. E. (2025). Genome-wide local ancestry and the
functional consequences of admixture in African and European cattle
populations. *Heredity*, *134*(1), 49–63.
doi:[10.1038/s41437-024-00734-w](https://doi.org/10.1038/s41437-024-00734-w).

</div>

<div id="ref-mchugoetal2024dryad" class="csl-entry">

**McHugo, G. P.**, Ward, J. A., Ng’ang’a, S. I., Frantz, L. A. F.,
Salter-Townshend, M., Hill, E. W., O’Gorman, G. M., Meade, K. G., Hall,
T. J., & MacHugh, D. E. (2024). *Data from: <span
class="nocase">Genome-wide</span> local ancestry and the functional
consequences of admixture in African and European cattle populations*
\[Data set\]. Dryad.
doi:[10.5061/dryad.w3r22810n](https://doi.org/10.5061/dryad.w3r22810n).

</div>

<div id="ref-mchugoetal2024researchsquare" class="csl-entry">

**McHugo, G. P.**, Ward, J. A., Ng’ang’a, S. I., Frantz, L. A. F.,
Salter-Townshend, M., Hill, E. W., O’Gorman, G. M., Meade, K. G., Hall,
T. J., & MacHugh, D. E. (2024). *Genome-wide local ancestry and the
functional consequences of admixture in African and European cattle
populations* \[Preprint\]. Research Square.
doi:[10.21203/rs.3.rs-4622059/v1](https://doi.org/10.21203/rs.3.rs-4622059/v1).

</div>

<div id="ref-mchugo2024zenodo2" class="csl-entry">

**McHugo, G. P.** (2024). *Code for Functional genomics of trypanotolerant
and trypanosusceptible cattle infected with Trypanosoma congolense
across multiple time points and tissues* \[Code\]. Zenodo.
doi:[10.5281/zenodo.11502110](https://doi.org/10.5281/zenodo.11502110).

</div>

<div id="ref-mchugo2024zenodo1" class="csl-entry">

**McHugo, G. P.** (2024). *Code for <span class="nocase">Genome-wide</span>
local ancestry and the functional consequences of admixture in African
and European cattle populations* \[Code\]. Zenodo.
doi:[10.5281/zenodo.11491949](https://doi.org/10.5281/zenodo.11491949).

</div>

<div id="ref-mchugo2024zenodo3" class="csl-entry">

**McHugo, G. P.** (2024). *Code for Integrative genomics of trypanotolerant
and trypanosusceptible hybrid African cattle using genomic and
transcriptomic data* \[Code\]. Zenodo.
doi:[10.5281/zenodo.11517979](https://doi.org/10.5281/zenodo.11517979).

</div>

<div id="ref-mchugo2024zenodo" class="csl-entry">

**McHugo, G. P.** (2024). *Supplementary material for Integrative population
and functional genomics of African cattle* \[Data set\]. Zenodo.
doi:[10.5281/zenodo.12205200](https://doi.org/10.5281/zenodo.12205200).

</div>

<div id="ref-mchugoetal2024biorxiv" class="csl-entry">

**McHugo, G. P.**, Ward, J. A., Ng’ang’a, S. I., Frantz, L. A. F.,
Salter-Townshend, M., Hill, E. W., O’Gorman, G. M., Meade, K. G., Hall,
T. J., & MacHugh, D. E. (2024). *Genome-wide local ancestry and the
functional consequences of admixture in African and European cattle
populations* \[Preprint\]. bioRxiv, 2024.06.20.599852.
doi:[10.1101/2024.06.20.599852](https://doi.org/10.1101/2024.06.20.599852).

</div>

<div id="ref-halletal2024tuberculosis" class="csl-entry">

Hall, T. J., **McHugo, G. P.**, Mullen, M. P., Ward, J. A., Killick, K. E.,
Browne, J. A., Gordon, S. V., & MacHugh, D. E. (2024). Integrative and
comparative genomic analyses of mammalian macrophage responses to
intracellular mycobacterial pathogens. *Tuberculosis*, *147*, 102453.
doi:[10.1016/j.tube.2023.102453](https://doi.org/10.1016/j.tube.2023.102453).

</div>

<div id="ref-kellyetal2024parasiteimmunology" class="csl-entry">

Kelly, P. A., **McHugo, G. P.**, Scaife, C., Peters, S., Stevenson, M. L.,
McKay, J. S., MacHugh, D. E., Saez, I. L., & Breathnach, R. (2024).
Unveiling the role of endoplasmic reticulum stress pathways in canine
demodicosis. *Parasite Immunology*, *46*(4), e13033.
doi:[10.1111/pim.13033](https://doi.org/10.1111/pim.13033).

</div>

<div id="ref-wardetal2024royalsocietyopenscience" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., Pérez O’Brien, A. M.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. E. (2024). Genomic insights
into the population history and adaptive traits of Latin American
Criollo cattle. *Royal Society Open Science*, *11*(3), 231388.
doi:[10.1098/rsos.231388](https://doi.org/10.1098/rsos.231388).

</div>

<div id="ref-wardetal2024royalsocietyopenscience1" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., O’Brien, A. M. P.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. (2024).
*Ward_et_al.(2023)\_Supp\_Mat\_REVISION from Genomic insights into the
population history and adaptive traits of Latin American Criollo cattle*
\[Data set\]. Royal Society Open Science.
doi:[10.6084/m9.figshare.25466153](https://doi.org/10.6084/m9.figshare.25466153).

</div>

<div id="ref-wardetal2024royalsocietyopenscience2" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., O’Brien, A. M. P.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. (2024).
*Ward_et_al.(2023)\_Table\_S2 from Genomic insights into the population
history and adaptive traits of Latin American Criollo cattle* \[Data
set\]. Royal Society Open Science.
doi:[10.6084/m9.figshare.25466159](https://doi.org/10.6084/m9.figshare.25466159).

</div>

<div id="ref-wardetal2024royalsocietyopenscience3" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., O’Brien, A. M. P.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. (2024).
*Ward_et_al.(2023)\_Table\_S3 from Genomic insights into the population
history and adaptive traits of Latin American Criollo cattle* \[Data
set\]. Royal Society Open Science.
doi:[10.6084/m9.figshare.25466150](https://doi.org/10.6084/m9.figshare.25466150).

</div>

<div id="ref-wardetal2024royalsocietyopenscience4" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., O’Brien, A. M. P.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. (2024).
*Ward_et_al.(2023)\_Table\_S8 from Genomic insights into the population
history and adaptive traits of Latin American Criollo cattle* \[Data
set\]. Royal Society Open Science.
doi:[10.6084/m9.figshare.25466156](https://doi.org/10.6084/m9.figshare.25466156).

</div>

<div id="ref-wardetal2024figshare" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., Pérez O’Brien, A. M.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. E. (2024). *Supplementary
material from Genomic insights into the population history and adaptive
traits of Latin American Criollo cattle* \[Collection\]. Figshare.
doi:[10.6084/m9.figshare.c.7146965](https://doi.org/10.6084/m9.figshare.c.7146965).

</div>

<div id="ref-ogradyetal2024biorxiv" class="csl-entry">

O’Grady, J. F., **McHugo, G. P.**, Ward, J. A., Hall, T. J., Faherty
O’Donnell, S. L., Correia, C. N., Browne, J. A., McDonald, M., Gormley,
E., Riggio, V., Prendergast, J. G. D., Clark, E. L., Pausch, H., Meade,
K. G., Gormley, I. C., Gordon, S. V., & MacHugh, D. E. (2024).
*Integrative genomics sheds light on the immunobiology of tuberculosis
in cattle* \[Preprint\]. bioRxiv, 2024.02.27.582295.
doi:[10.1101/2024.02.27.582295](https://doi.org/10.1101/2024.02.27.582295).

</div>

<div id="ref-ogradyetal2024geneexpressionomnibus" class="csl-entry">

O’Grady, J. F., **McHugo, G. P.**, Ward, J. A., Hall, T. J., Faherty
O’Donnell, S. L., Correia, C. N., Browne, J. A., McDonald, M., Gormley,
E., Riggio, V., Prendergast, J. G. D., Clark, E. L., Pausch, H., Meade,
K. G., Gormley, I. C., Gordon, S. V., & MacHugh, D. E. (2024).
*Integrative genomics sheds light on the immunobiology of tuberculosis
in cattle* \[Data set\]. Gene Expression Omnibus.
[https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE255724.](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE255724)

</div>

<div id="ref-ogradyetal2024zenodo" class="csl-entry">

O’Grady, J. F., **McHugo, G. P.**, Ward, J. A., Hall, T. J., Faherty
O’Donnell, S. L., Correia, C. N., Browne, J. A., McDonald, M., Gormley,
E., Riggio, V., Prendergast, J. G. D., Clark, E. L., Pausch, H., Meade,
K. G., Gormley, I. C., Gordon, S. V., & MacHugh, D. E. (2024).
*Integrative genomics sheds light on the immunobiology of tuberculosis
in cattle* \[Data set\]. Zenodo.
doi:[10.5281/zenodo.10658452](https://doi.org/10.5281/zenodo.10658452).

</div>

<div id="ref-wardetal2023europeannucleotidearchive" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., Pérez O’Brien, A. M.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. E. (2023). *Population
genetics of Latin American Criollo cattle* \[Data set\]. European
Nucleotide Archive.
[https://www.ebi.ac.uk/ena/browser/view/PRJEB65887.](https://www.ebi.ac.uk/ena/browser/view/PRJEB65887)

</div>

<div id="ref-bhatetal2023frontiersinimmunology1" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *DataSheet\_1\_Preferential
differential gene expression within the WC1.1+ γδ T cell compartment in
cattle naturally infected with Mycobacterium bovis.xlsx* \[Data set\].
Frontiers in Immunology.
doi:[10.3389/fimmu.2023.1265038.s001](https://doi.org/10.3389/fimmu.2023.1265038.s001).

</div>

<div id="ref-bhatetal2023frontiersinimmunology2" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *DataSheet\_2\_Preferential
differential gene expression within the WC1.1+ γδ T cell compartment in
cattle naturally infected with Mycobacterium bovis.xlsx* \[Data set\].
Frontiers in Immunology.
doi:[10.3389/fimmu.2023.1265038.s002](https://doi.org/10.3389/fimmu.2023.1265038.s002).

</div>

<div id="ref-bhatetal2023frontiersinimmunology3" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *DataSheet\_3\_Preferential
differential gene expression within the WC1.1+ γδ T cell compartment in
cattle naturally infected with Mycobacterium bovis.xlsx* \[Data set\].
Frontiers in Immunology.
doi:[10.3389/fimmu.2023.1265038.s003](https://doi.org/10.3389/fimmu.2023.1265038.s003).

</div>

<div id="ref-bhatetal2023frontiersinimmunology4" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *DataSheet\_4\_Preferential
differential gene expression within the WC1.1+ γδ T cell compartment in
cattle naturally infected with Mycobacterium bovis.xlsx* \[Data set\].
Frontiers in Immunology.
doi:[10.3389/fimmu.2023.1265038.s004](https://doi.org/10.3389/fimmu.2023.1265038.s004).

</div>

<div id="ref-bhatetal2023frontiersinimmunology5" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *DataSheet\_5\_Preferential
differential gene expression within the WC1.1+ γδ T cell compartment in
cattle naturally infected with Mycobacterium bovis.xlsx* \[Data set\].
Frontiers in Immunology.
doi:[10.3389/fimmu.2023.1265038.s005](https://doi.org/10.3389/fimmu.2023.1265038.s005).

</div>

<div id="ref-bhatetal2023frontiersinimmunology6" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *DataSheet\_6\_Preferential
differential gene expression within the WC1.1+ γδ T cell compartment in
cattle naturally infected with Mycobacterium bovis.xlsx* \[Data set\].
Frontiers in Immunology.
doi:[10.3389/fimmu.2023.1265038.s006](https://doi.org/10.3389/fimmu.2023.1265038.s006).

</div>

<div id="ref-bhatetal2023frontiersinimmunology" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). Preferential differential gene
expression within the WC1.1+ γδ T cell compartment in cattle naturally
infected with Mycobacterium bovis. *Frontiers in Immunology*, *14*,
1265038.
doi:[10.3389/fimmu.2023.1265038](https://doi.org/10.3389/fimmu.2023.1265038).

</div>

<div id="ref-wardetal2023biorxiv" class="csl-entry">

Ward, J. A., Ng’ang’a, S. I., Randhawa, I. A. S., **McHugo, G. P.**,
O’Grady, J. F., Flórez, J. M., Browne, J. A., Pérez O’Brien, A. M.,
Landaeta-Hernández, A. J., Garcia, J. F., Sonstegard, T. S., Frantz, L.
A. F., Salter-Townshend, M., & MacHugh, D. E. (2023). *Genomic insights
into the population history and adaptive traits of Latin American
Criollo cattle* \[Preprint\]. bioRxiv, 2023.09.10.556512.
doi:[10.1101/2023.09.10.556512](https://doi.org/10.1101/2023.09.10.556512).

</div>

<div id="ref-kellyetal2023biorxiv" class="csl-entry">

Kelly, P. A., **McHugo, G. P.**, Scaife, C., Peters, S., Stevenson, M. L.,
McKay, J. S., MacHugh, D. E., Lara-Saez, I., & Breathnach, R. (2023).
*Unveiling the role of endoplasmic reticulum stress pathways in canine
demodicosis* \[Preprint\]. bioRxiv, 2023.08.11.552979.
doi:[10.1101/2023.08.11.552979](https://doi.org/10.1101/2023.08.11.552979).

</div>

<div id="ref-bhatetal2023biorxiv" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *Preferential differential gene
expression within the WC1.1+ γδ T cell compartment in cattle naturally
infected with Mycobacterium bovis* \[Preprint\]. bioRxiv,
2023.07.21.550071.
doi:[10.1101/2023.07.21.550071](https://doi.org/10.1101/2023.07.21.550071).

</div>

<div id="ref-halletal2023biorxiv" class="csl-entry">

Hall, T. J., **McHugo, G. P.**, Mullen, M. P., Ward, J. A., Killick, K. E.,
Browne, J. A., Gordon, S. V., & MacHugh, D. E. (2023). *Integrative and
comparative genomic analyses of mammalian macrophage responses to
intracellular mycobacterial pathogens* \[Preprint\]. bioRxiv,
2023.07.14.549042.
doi:[10.1101/2023.07.14.549042](https://doi.org/10.1101/2023.07.14.549042).

</div>

<div id="ref-bhatetal2023biostudies" class="csl-entry">

Bhat, S. A., Elnaggar, M., Hall, T. J., **McHugo, G. P.**, Reid, C.,
MacHugh, D. E., & Meade, K. G. (2023). *Preferential differential gene
expression in the WC1.1+ γδ T cell compartment in cattle naturally
infected with Mycobacterium bovis* \[Data set\]. BioStudies.
[https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-13111.](https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-13111)

</div>

<div id="ref-correiaetal2022tuberculosis" class="csl-entry">

Correia, C. N., **McHugo, G. P.**, Browne, J. A., McLoughlin, K. E., Nalpas,
N. C., Magee, D. A., Whelan, A. O., Villarreal-Ramos, B., Vordermeier,
H. M., Gormley, E., Gordon, S. V., & MacHugh, D. E. (2022).
High-resolution transcriptomics of bovine purified protein
derivative-stimulated peripheral blood from cattle infected with
*Mycobacterium bovis* across an experimental time course. *Tuberculosis*,
*136*, 102235.
doi:[10.1016/j.tube.2022.102235](https://doi.org/10.1016/j.tube.2022.102235).

</div>

<div id="ref-correiaetal2022europeannucleotidearchive"
class="csl-entry">

Correia, C. N., **McHugo, G. P.**, Browne, J. A., McLoughlin, K. E., Nalpas,
N. C., Magee, D. A., Whelan, A. O., Villarreal-Ramos, B., Vordermeier,
H. M., Gormley, E., Gordon, S. V., & MacHugh, D. E. (2022). *<span
class="nocase">RNA-seq</span> transcriptional profile of <span
class="nocase">PPD-b</span> stimulated peripheral blood from cattle
infected with M. bovis across an experimental time course* \[Data set\].
European Nucleotide Archive.
[https://www.ebi.ac.uk/ena/browser/view/PRJEB44568.](https://www.ebi.ac.uk/ena/browser/view/PRJEB44568)

</div>

<div id="ref-wardetal2022mendeleydata" class="csl-entry">

Ward, J. A., **McHugo, G. P.**, Dover, M. J., Hall, T. J., Ng’ang’a, S. I.,
Sonstegard, T. S., Bradley, D. G., Frantz, L. A. F., Salter-Townshend,
M., & MacHugh, D. E. (2022). *Genome-wide local ancestry and evidence
for mitonuclear co-adaptation in African hybrid cattle populations (Bos
taurus/indicus). Ward et al. 2022* \[Data set\]. Mendeley Data.
doi:[10.17632/yt3tgpt48d](https://doi.org/10.17632/yt3tgpt48d).

</div>

<div id="ref-wardetal2022iscience" class="csl-entry">

Ward, J. A., **McHugo, G. P.**, Dover, M. J., Hall, T. J., Ng’ang’a, S. I.,
Sonstegard, T. S., Bradley, D. G., Frantz, L. A. F., Salter-Townshend,
M., & MacHugh, D. E. (2022). Genome-wide local ancestry and evidence for
mitonuclear coadaptation in African hybrid cattle populations.
*iScience*, *25*(7), 104672.
doi:[10.1016/j.isci.2022.104672](https://doi.org/10.1016/j.isci.2022.104672).

</div>

<div id="ref-correiaetal2022biorxiv" class="csl-entry">

Correia, C. N., **McHugo, G. P.**, Browne, J. A., McLoughlin, K. E., Nalpas,
N. C., Magee, D. A., Whelan, A. O., Villarreal-Ramos, B., Vordermeier,
H. M., Gormley, E., Gordon, S. V., & MacHugh, D. E. (2022).
*High-resolution transcriptomics of bovine purified protein
derivative-stimulated peripheral blood from cattle infected with
Mycobacterium bovis across an experimental time course* \[Preprint\].
bioRxiv, 2022.04.19.488769.
doi:[10.1101/2022.04.19.488769](https://doi.org/10.1101/2022.04.19.488769).

</div>

<div id="ref-wardetal2022ssrn" class="csl-entry">

Ward, J. A., **McHugo, G. P.**, Dover, M. J., Hall, T. J., Ng’ang’a, S. I.,
Sonstegard, T. S., Bradley, D. G., Frantz, L. A. F., Salter-Townshend,
M., & MacHugh, D. E. (2022). *Genome-wide local ancestry and direct
evidence for mitonuclear coadaptation in African hybrid cattle
populations (Bos taurus/indicus)* \[Preprint\]. SSRN.
doi:[10.2139/ssrn.4055643](https://doi.org/10.2139/ssrn.4055643).

</div>

<div id="ref-wardetal2021biorxiv" class="csl-entry">

Ward, J. A., **McHugo, G. P.**, Dover, M. J., Hall, T. J., Ng’ang’a, S. I.,
Sonstegard, T. S., Bradley, D. G., Frantz, L. A. F., Salter-Townshend,
M., & MacHugh, D. E. (2021). *Genome-wide local ancestry and direct
evidence for mitonuclear co-adaptation in African hybrid cattle
populations (Bos taurus/indicus)* \[Preprint\]. bioRxiv,
2021.08.26.457829.
doi:[10.1101/2021.08.26.457829](https://doi.org/10.1101/2021.08.26.457829).

</div>

<div id="ref-doveretal2021irishjournalofmedicalscience"
class="csl-entry">

Dover, M. J., Hall, T. J., **McHugo, G. P.**, & MacHugh, D. E. (2021). A
functional and integrative genomics analysis of host response to disease
caused by infection with *Mycobacterium tuberculosis* in humans and
*Mycobacterium bovis* in cattle \[Abstract\]. 2020 UCD School of Medicine
Summer Student Research Awards. *Irish Journal of Medical Science*,
*190*(Supplement 4), S135–S136.
doi:[10.1007/s11845-021-02718-9](https://doi.org/10.1007/s11845-021-02718-9).

</div>

<div id="ref-halletal2021bmcgenomics1" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 1 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583719](https://doi.org/10.6084/m9.figshare.14583719).

</div>

<div id="ref-halletal2021bmcgenomics2" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 2 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583722](https://doi.org/10.6084/m9.figshare.14583722).

</div>

<div id="ref-halletal2021bmcgenomics3" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 3 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583725](https://doi.org/10.6084/m9.figshare.14583725).

</div>

<div id="ref-halletal2021bmcgenomics4" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 4 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583728](https://doi.org/10.6084/m9.figshare.14583728).

</div>

<div id="ref-halletal2021bmcgenomics5" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 5 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583731](https://doi.org/10.6084/m9.figshare.14583731).

</div>

<div id="ref-halletal2021bmcgenomics6" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 6 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583734](https://doi.org/10.6084/m9.figshare.14583734).

</div>

<div id="ref-halletal2021bmcgenomics7" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 7 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583740](https://doi.org/10.6084/m9.figshare.14583740).

</div>

<div id="ref-halletal2021bmcgenomics8" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Additional file 8 of Integrative genomics of the
mammalian alveolar macrophage response to intracellular mycobacteria*
\[Data set\]. BMC Genomics.
doi:[10.6084/m9.figshare.14583743](https://doi.org/10.6084/m9.figshare.14583743).

</div>

<div id="ref-halletal2021figshare" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). *Integrative genomics of the mammalian alveolar macrophage
response to intracellular mycobacteria* \[Collection\]. Figshare.
doi:[10.6084/m9.figshare.c.5422454](https://doi.org/10.6084/m9.figshare.c.5422454).

</div>

<div id="ref-halletal2021bmcgenomics" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2021). Integrative genomics of the mammalian alveolar macrophage
response to intracellular mycobacteria. *BMC Genomics*, *22*, 343.
doi:[10.1186/s12864-021-07643-w](https://doi.org/10.1186/s12864-021-07643-w).

</div>

<div id="ref-halletal2021dryad" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Berry, D. P.,
Ring, S. C., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh, D.
E. (2021). *Integrative genomics of the mammalian alveolar macrophage
response to intracellular mycobacteria: <span
class="nocase">RNA-seq</span> statistics and results* \[Data set\].
Dryad.
doi:[10.5061/dryad.83bk3j9q6](https://doi.org/10.5061/dryad.83bk3j9q6).

</div>

<div id="ref-sauraetal2020frontiers" class="csl-entry">

Saura, M., Bertolini, F., & Palaiokostas, C. (Eds.). (2020). *Genetic
variability in conservation and selection programs in the post-genomics
era* \[Research topic\]. Frontiers.
doi:[10.3389/978-2-88966-249-4](https://doi.org/10.3389/978-2-88966-249-4).

</div>

<div id="ref-halletal2020researchsquare" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2020). *Integrative genomics of the mammalian alveolar macrophage
response to intracellular mycobacteria* \[Preprint\]. Research Square.
doi:[10.21203/rs.3.rs-121955/v1](https://doi.org/10.21203/rs.3.rs-121955/v1).

</div>

<div id="ref-halletal2020biorxiv" class="csl-entry">

Hall, T. J., Mullen, M. P., **McHugo, G. P.**, Killick, K. E., Ring, S. C.,
Berry, D. P., Correia, C. N., Browne, J. A., Gordon, S. V., & MacHugh,
D. E. (2020). *Integrative genomics of the mammalian alveolar macrophage
response to intracellular mycobacteria* \[Preprint\]. bioRxiv,
2020.08.25.266668.
doi:[10.1101/2020.08.25.266668](https://doi.org/10.1101/2020.08.25.266668).

</div>

<div id="ref-mchugoetal2019bmcbiology" class="csl-entry">

**McHugo, G. P.**, Dover, M. J., & MacHugh, D. E. (2019). Unlocking the
origins and biology of domestic animals using ancient DNA and
paleogenomics. *BMC Biology*, *17*, 98.
doi:[10.1186/s12915-019-0724-7](https://doi.org/10.1186/s12915-019-0724-7).

</div>

<div id="ref-mchugoetal2019frontiersingenetics" class="csl-entry">

**McHugo, G. P.**, Browett, S., Randhawa, I. A. S., Howard, D. J., Mullen,
M. P., Richardson, I. W., Park, S. D. E., Magee, D. A., Scraggs, E.,
Dover, M. J., Correia, C. N., Hanrahan, J. P., & MacHugh, D. E. (2019).
A population genomics analysis of the native Irish Galway sheep breed.
*Frontiers in Genetics*, *10*, 927.
doi:[10.3389/fgene.2019.00927](https://doi.org/10.3389/fgene.2019.00927).

</div>

<div id="ref-mchugoetal2019frontiersingenetics1" class="csl-entry">

**McHugo, G. P.**, Browett, S., Randhawa, I. A. S., Howard, D. J., Mullen,
M. P., Richardson, I. W., Park, S. D. E., Magee, D. A., Scraggs, E.,
Dover, M. J., Correia, C. N., Hanrahan, J. P., & MacHugh, D. E. (2019).
*A population genomics analysis of the native Irish Galway sheep breed:
Table\_1.docx* \[Data set\]. Frontiers in Genetics.
doi:[10.3389/fgene.2019.00927.s001](https://doi.org/10.3389/fgene.2019.00927.s001).

</div>

<div id="ref-halletal2019researchgate" class="csl-entry">

Hall, T. J., Mullen, M. P., Killick, K. E., **McHugo, G. P.**, Browne, J.
A., Correia, C. N., Gordon, S. V., & MacHugh, D. E. (2019). *Integrative
genomics of the host response to Mycobacterium bovis in bovine alveolar
macrophages* \[Poster\]. 37th International Society for Animal Genetics
Conference. ResearchGate.
doi:[10.13140/rg.2.2.17194.36807](https://doi.org/10.13140/rg.2.2.17194.36807).

</div>

<div id="ref-mchugoetal2019biorxiv" class="csl-entry">

**McHugo, G. P.**, Browett, S., Randhawa, I. A. S., Howard, D. J., Mullen,
M. P., Richardson, I. W., Park, S. D. E., Magee, D. A., Scraggs, E.,
Dover, M., Correia, C. N., Hanrahan, J. P., & MacHugh, D. E. (2019). *A
population genomics analysis of the native Irish Galway sheep breed*
\[Preprint\]. bioRxiv, 645754.
doi:[10.1101/645754](https://doi.org/10.1101/645754).

</div>

<div id="ref-mchugoetal2019humanheredity" class="csl-entry">

**McHugo, G. P.**, O’Gorman, G. M., Hill, E. W., & MacHugh, D. E. (2019).
Local ancestry and functional genomics of trypanotolerant and
trypanosusceptible admixed African cattle breeds \[Abstract\]. 47th
European Mathematical Genetics Meeting. *Human Heredity*, *83*(5), 250.
doi:[10.1159/000499459](https://doi.org/10.1159/000499459).

</div>

<div id="ref-browettetal2019dryad" class="csl-entry">

Browett, S., **McHugo, G.**, Richardson, I. W., Magee, D. A., Park, S. D.
E., Fahey, A. G., Kearney, J. F., Correia, C. N., Randhawa, I. A. S., &
MacHugh, D. E. (2019). *Data from: Genomic characterisation of the
indigenous Irish Kerry cattle breed* \[Data set\]. Dryad.
doi:[10.5061/dryad.8fk81](https://doi.org/10.5061/dryad.8fk81).

</div>

<div id="ref-browettetal2018frontiersingenetics" class="csl-entry">

Browett, S., **McHugo, G.**, Richardson, I. W., Magee, D. A., Park, S. D.
E., Fahey, A. G., Kearney, J. F., Correia, C. N., Randhawa, I. A. S., &
MacHugh, D. E. (2018). Genomic characterisation of the indigenous Irish
Kerry cattle breed. *Frontiers in Genetics*, *9*, 51.
doi:[10.3389/fgene.2018.00051](https://doi.org/10.3389/fgene.2018.00051).

</div>

<div id="ref-browettetal2018frontiersingenetics1" class="csl-entry">

Browett, S., **McHugo, G.**, Richardson, I. W., Magee, D. A., Park, S. D.
E., Fahey, A. G., Kearney, J. F., Correia, C. N., Randhawa, I. A. S., &
MacHugh, D. E. (2018). *Genomic characterisation of the indigenous Irish
Kerry cattle breed: DataSheet1.pdf* \[Data set\]. Frontiers in Genetics.
doi:[10.3389/fgene.2018.00051.s001](https://doi.org/10.3389/fgene.2018.00051.s001).

</div>

<div id="ref-browettetal2018frontiersingenetics2" class="csl-entry">

Browett, S., **McHugo, G.**, Richardson, I. W., Magee, D. A., Park, S. D.
E., Fahey, A. G., Kearney, J. F., Correia, C. N., Randhawa, I. A. S., &
MacHugh, D. E. (2018). *Genomic characterisation of the indigenous Irish
Kerry cattle breed: Table1.xlsx* \[Data set\]. Frontiers in Genetics.
doi:[10.3389/fgene.2018.00051.s002](https://doi.org/10.3389/fgene.2018.00051.s002).

</div>

<div id="ref-browettetal2018frontiersingenetics3" class="csl-entry">

Browett, S., **McHugo, G.**, Richardson, I. W., Magee, D. A., Park, S. D.
E., Fahey, A. G., Kearney, J. F., Correia, C. N., Randhawa, I. A. S., &
MacHugh, D. E. (2018). *Genomic characterisation of the indigenous Irish
Kerry cattle breed: Table6.xlsx* \[Data set\]. Frontiers in Genetics.
doi:[10.3389/fgene.2018.00051.s003](https://doi.org/10.3389/fgene.2018.00051.s003).

</div>

<div id="ref-browettetal2017biorxiv" class="csl-entry">

Browett, S., **McHugo, G.**, Richardson, I. W., Magee, D. A., Park, S. D.
E., Fahey, A. G., Kearney, J. F., Correia, C. N., Randhawa, I. A. S., &
MacHugh, D. E. (2017). *Genomic characterisation and conservation
genetics of the indigenous Irish Kerry cattle breed* \[Preprint\].
bioRxiv, 210229. doi:[10.1101/210229](https://doi.org/10.1101/210229).

</div>

</div>
